File: platforms/kotlin/examples/core/texture_mip_chain/format.kt
import org.fragmentcolor.*

val pixels = Array(4 * 4 * 4) { 200 }
val chain = TextureMipChain.prepare((
    pixels.asSlice(),
    TextureFormat.Rgba8UnormSrgb,
    arrayOf(4, 4),
))
val _ = chain.format()

File: platforms/swift/examples/core/texture_mip_chain/format.swift
import FragmentColor

let pixels = Array(repeating: 200, count: 4 * 4 * 4)
let chain = try TextureMipChain.prepare((
    pixels,
    TextureFormat.rgba8UnormSrgb,
    [4, 4],
))
let _ = chain.format()